app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.ts

```diff
@@ -249,11 +249,28 @@ export class AwsLoadBalancerTransformer {
             action.targetGroupName = action.targetGroupName.replace(`${applicationName}-`, '');
           });
 
+          // Remove the default rule because it already exists in defaultActions
+          listener.rules = (listener.rules || []).filter((l) => !l.default);
+          listener.rules.forEach((rule) => {
+            (rule.actions || []).forEach((action) => {
+              action.targetGroupName = action.targetGroupName.replace(`${applicationName}-`, '');
+            });
+            rule.conditions = rule.conditions || [];
+          });
+
+          // Sort listener.rules by priority.
+          listener.rules.sort((a, b) => {
+            if (a.priority < b.priority) { return -1; }
+            if (a.priority > b.priority) { return 1; }
+            return 0;
```

Comment:
Yes, it can. I was taking into account typescript not liking subtracting a `'default'` from a `number`, but casting the priority to a number works (acceptable because i remove the rule that has a priority of `'default'`) `(a.priority as number) - (b.priority as number)`